data-processing-lib/python/src/data_processing/data_access/data_access_local.py

```diff
@@ -260,8 +260,9 @@ def get_table(self, path: str) -> tuple[pa.table, int]:
         Args:
             path (str): Path to the file containing the table.
 
-        Returns:
+        Returns: Tuple containing
             pyarrow.Table: PyArrow table if read successfully, None otherwise.
+            the number of retries.
         """
 
         try:
```

Comment:
We already have it https://github.com/IBM/data-prep-kit/blob/dev/data-processing-lib/python/src/data_processing/data_access/data_access_local.py#L256